agent/consul/state/coordinate.go

```diff
@@ -42,21 +42,24 @@ func (s *Restore) Coordinates(idx uint64, updates structs.Coordinates) error {
 
 // Coordinate returns a map of coordinates for the given node, indexed by
 // network segment.
-func (s *Store) Coordinate(node string) (lib.CoordinateSet, error) {
+func (s *Store) Coordinate(node string, ws memdb.WatchSet) (uint64, lib.CoordinateSet, error) {
```

Comment:
Would be good to add a little coverage for `ws` and the returned index in the unit tests.